app/backend/user/views.py

```
@@ -312,9 +341,38 @@ def get_queryset(self):
 class UnbanUserView(APIView):
     permission_classes = (permissions.IsAuthenticated, IsUserAdmin)
 
-    def delete(self, request, user_id):
+    def restore_blacklisted_user(self, user_id):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)